action.yml

```diff
@@ -51,9 +55,11 @@ runs:
           echo "unknown platform $platform"
           exit 1
         }
+        $vol = '${{ inputs.work-vol }}'
```

Comment:
Perhaps some bullet-proofing here?

If it's one character, append a ':'?
If it's exactly two characters it must end with a ':'
I guess theoretically this could be a UNC or long path name, so otherwise it must start '\\'

Or just an error if it's not of the expected form?

Reply:
If it is a bad path prefix, download of setup.exe will fail soon after.  Is that not sufficient?

BTW, there are no such bullet-proofing checks on `install-dir`.

Reply:
Yeah, this is probably excessive.

Maybe if someone mistakenly uses 'C' rather than 'C:' you get a directory named C in cwd, or something, but I guess they'll learn not to do that 😖